.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,160 @@
+name: benchmarks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled # run-benchmarks label required
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build:
+    if: github.event_name == 'push' ||
+        github.event_name == 'workflow_dispatch' || 
+        contains(github.event.pull_request.labels.*.name, 'run-benchmarks')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Download Atom Data
+        run: |
+          REPO_URL="https://dev.azure.com/tardis-sn/TARDIS/_apis/git/repositories/tardis-refdata"
+          wget -q "$REPO_URL/items?path=atom_data/kurucz_cd23_chianti_H_He.h5&versionType=branch&version=master&resolveLfs=true" -O benchmarks/data/kurucz_cd23_chianti_H_He.h5
+
+      - name: Install asv
+        run: pip install asv
```

Comment:
Some pinning would be ok, right? at least major-minor

Reply:
I changed it over to 0.5 in https://github.com/tardis-sn/tardis/pull/2260/commits/a0b33f43c650f6301be8bf80d9fd092e6eae5027. The 0.3 version is very old(2018) and this version(0.5) was released nearly a year ago(Feb 2022). The 0.3 version causes environment errors in the pipeline.